config-overrides.js

```diff
@@ -1,4 +1,16 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-const { useBabelRc, override } = require('customize-cra');
+const { useBabelRc, override, addWebpackAlias } = require('customize-cra');
+const path = require('path');
 
-module.exports = override(useBabelRc());
+module.exports = override(
+  useBabelRc(),
+  addWebpackAlias({
+    '@Constants': path.resolve(__dirname, './src/constants'),
+    '@SharedComponents': path.resolve(__dirname, './src/components'),
+    '@Services': path.resolve(__dirname, './src/services'),
+    '@Pages': path.resolve(__dirname, './src/pages'),
+    '@HOCs': path.resolve(__dirname, './src/HOCs'),
+    '@Images': path.resolve(__dirname, './src/images'),
+    '@Src': path.resolve(__dirname, './src'),
```

Comment:
I think there are one or two places where we import something from src itself (so like src/contexts.js) so that's why I put it there. I can remove it though.

Reply:
Nvm saw your other comment

Reply:
yeah I just think it makes a lil more sense because all of our @ aliases are all towards an actual folder that represents a collection of entities - would love to hear it if you have another idea

Reply:
No I think @Contexts is fine. We don't really have any other file in ./src to export, so I think @Contexts is better and more specific.